src/Microsoft.OpenApi.Readers/ParseNodes/ParseNode.cs

```diff
@@ -113,5 +114,11 @@ public virtual string GetScalarValue()
         {
             throw new OpenApiException("Cannot get scalar value");
         }
+
+        public virtual List<IOpenApiAny> CreateListOfAny()
+        {
+            throw new OpenApiException("Cannot create list of any from here");
```

Comment:
Maybe it's better to throw "NotSupported"

Reply:
It definitely could use a better description.  However, I want this error to be aggregated in the errors collection.

Reply:
I've cleaned up the descriptions a bit so they are consistent.